webpack/assets/javascripts/services/ChartService.js

```diff
@@ -1,10 +1,15 @@
 import uuidV1 from 'uuid/v1';
 import Immutable from 'seamless-immutable';
-import { donutChartConfig, donutLargeChartConfig } from './ChartService.consts';
-
-const sizeConfig = {
-  regular: donutChartConfig,
-  large: donutLargeChartConfig,
+import {
+  donutChartConfig,
+  donutLargeChartConfig,
+  barChartConfig,
+} from './ChartService.consts';
+
+const configForType = {
+  regularDonut: donutChartConfig,
+  largeDonut: donutLargeChartConfig,
+  regularBar: barChartConfig,
```

Comment:
IMHO, since there is only one size (`regular`) to `BarChart`, it is not a good idea to add it to the `configForType` (previously `sizeConfig`) object.

If we insist on doing that, maybe keep the name `sizeConfig` and `nested it`:

```javascript
const sizeConfig = {
  donut: {
    regular: ...
    large: ...
  },
  bar: {
    ...
  }
}

const getChartConfig = ({
  ...
  type,
  config
}) = {
  chartConfigForType = sizeConfig[type][config]
  ...
}
```